app/javascript/components/pages/AboutUs.js

```diff
@@ -1,7 +1,28 @@
 import React from "react";
 
 const AboutUs = () => {
-  return <div>AboutUs</div>;
+  return(
+  <>
+    <div>ARCT-TECH</div>
+    <div>A team of Didja-dooers, here to help you achieve your goals!</div>
+    <div>
+      <img src="https://avatars.githubusercontent.com/u/111712184?s=400&u=1d5dedbbaa52cfbd69091cbbb02e84080298ae7e&v=4" />
+      Anthony: (Anthony's Bio)
```

Comment:
I would like to see these bios filled out on this branch so that you can mark this card as complete.  Does not need to be long and extravagant, just start with a quick sentence to describe yourself and your role on this project.  You can make updates later, but you really should provide some content here.